docs/change-log.md

```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update April 7th, 2023
+
+- **Improved:** [Driver inputs](https://docs.humanitec.com/reference/concepts/resources/definitions#driver-inputs) are now validated against Driver Definition Schemas to catch misconfiguration errors in [resource definitions](https://docs.humanitec.com/reference/concepts/resources/definitions) earlier.
+- **Fixed:** Special character "*" in [matching criteria](https://docs.humanitec.com/reference/concepts/resources/definitions#matching-criteria) added via API is now correctly interpreted as "all values".
```

Comment:
```suggestion
- **Fixed:** Selecting an asterisk `*` as a [matching criterium](https://api-docs.humanitec.com/#tag/MatchingCriteria) via the API  is now correctly interpreted as "all values".
```

I'm not sure the "when" works well with "it is interpreted" structure. How about this?

Reply:
Looks good :)